src/_includes/site_nav.hbs

```diff
@@ -0,0 +1,20 @@
+<nav class="site-nav headroom">
+  <div class="content">
+    <div class="sub-nav-left">
+      <a href="/"><img class="logo" src="/img/logos/localhost.png"/></a>
+      <div class="sub-nav-control visible-md down"></div>
+    </div>
+    <div class="hidden-md hello">
+      <ul class="sub-nav-right">
```

Comment:
if possible, give the parent div the class sub-nav-right so both sub-nav's start at the same indent point, easier on the eye: 

`<div class="sub-nav-right hidden-md hello">
    <ul class="nav-items">`